jsconfig.json

```diff
@@ -1,14 +1,25 @@
 {
-  "include": ["*.js", "./src/**/*.jsx"],
-  "exclude": ["node_modules", "./src/**/*.stories.*"],
+  "$schema": "https://json.schemastore.org/tsconfig",
+  "include": ["*.js", "./src/**/*", "./.storybook/**/*"],
+  "exclude": ["node_modules"],
   "compilerOptions": {
+    "paths": {
+      "$particles": ["./src/lib/components/particles"],
+      "$particles/*": ["./src/lib/components/particles/*"],
+      "$molecules": ["./src/lib/components/molecules"],
+      "$molecules/*": ["./src/lib/components/molecules/*"],
+      "$shared/*": ["./src/lib/shared/*"],
+      "$styles/*": ["./src/lib/styles/*"],
+      "$storybook/*": ["./.storybook/*"]
+    },
     "module": "es2022",
     "esModuleInterop": true,
     "target": "ESNext",
     "jsx": "react-jsx",
     "jsxImportSource": "preact",
     "jsxFactory": "createElement",
     "skipLibCheck": true,
-    "checkJs": false
+    "checkJs": true,
+    "allowJs": true
```

Comment:
Nice work on this 👍 

Minor thing: `allowJS` is true by default in jsconfig

![Screenshot 2024-08-07 at 08 11 26](https://github.com/user-attachments/assets/b71634ef-8726-4664-984d-139343fb5b93)

https://code.visualstudio.com/docs/languages/jsconfig

Reply:
Ah good to know, thanks!